addons/crm/report/crm_opportunity_report_views.xml

```diff
@@ -52,35 +52,6 @@
             </field>
         </record>
 
-        <!-- Custom reports (aka filters) -->
-        <record id="filter_opportunity_opportunities_cohort" model="ir.filters">
```

Comment:
Don't forget to explain in the commit message those have been removed on purpose :) .